ci/scripts/wait-for-deploy.js

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+
+const AWS = require("aws-sdk")
+const ecs = new AWS.ECS()
+const MAX_RETRIES = 120
+const CHECK_INTERVAL = 5000
+
+const describeServices = async function describeServices() {
+  var params = {
+    services: ['toolbox'],
+    cluster: 'test-12-fargate'
+  };
+  return await ecs.describeServices(params).promise()
+}
+
+const run = async function run() {
+  let uncompletedDeployments
+  let counter = 0
+  const deploymentChecker = setInterval(async () => {
+    counter++
+    if (counter === MAX_RETRIES) {
+      console.log(`Deployment did not complete after ${MAX_RETRIES*CHECK_INTERVAL} seconds.`)
+      process.exitCode = 1
+      clearInterval(deploymentChecker)
+    }
+    const data = await describeServices()
+    uncompletedDeployments = data.services[0].deployments
+      .filter(deployment => deployment.rolloutState !== 'COMPLETED')
+    if (uncompletedDeployments.length === 0) {
+      console.log('Deployment successful')
+      clearInterval(deploymentChecker)
+    } else {
+      const {taskDefinition, rolloutState, desiredCount, pendingCount, runningCount} = uncompletedDeployments[0]
+      console.log(
+        `Deploying task definition: ${taskDefinition}
+         Current status: ${rolloutState}
+         Desired count: ${desiredCount}
+         Pending count: ${pendingCount}
+         Running count: ${runningCount}`
```

Comment:
I think you said ignore the formatting so this is just a passing comment. I felt like at present Jenkins keeps printing the same line when it hasn't changed and its just noise and scrolling through it until you find the changes. Wonder if you could just print checking and the number of tries unless something has changed?

Reply:
yeah probably a good shout